internal/config/config_test.go

```
@@ -235,3 +181,62 @@ func TestInitConfig_WatchdogIntervalTooLarge(t *testing.T) {
 		t.Fatal("InitConfig() unexpectedly succeeded")
 	}
 }
+
+func TestRefreshAllowLists(t *testing.T) {
+	socketPath := filepath.Join(t.TempDir(), "docker.sock")
+	listener, err := net.Listen("unix", socketPath)
```

golangci-lint (2.12.2)

[error] net.Listen must not be called. use (*net.ListenConfig).Listen

(noctx)